tests/uri.cpp

```diff
@@ -38,14 +38,6 @@ BOOST_AUTO_TEST_CASE(host_port)
     BOOST_CHECK_EQUAL( hostPort.getScheme(), "tcp" );
 }
 
-BOOST_AUTO_TEST_CASE(custom_schema_not_supported_yet)
```

Comment:
// TODO: only for compatibility for hbp schemas, remove after downstreams		
// are fixed
I removed this to do from uri.cpp. What is in mind for this then ?